table.csv

```diff
@@ -73,6 +73,7 @@ ipns-ns,                        namespace,      0xe5,           IPNS path
 zeronet,                        namespace,      0xe6,           ZeroNet site address
 secp256k1-pub,                  key,            0xe7,           Secp256k1 public key
 ed25519-pub,                    key,            0xed,           Ed25519 public key
+x25519-pub,                     key,            0xec,           Curve25519 public key
```

Comment:
Out-of-order fix:

```suggestion
x25519-pub,                     key,            0xec,           Curve25519 public key
ed25519-pub,                    key,            0xed,           Ed25519 public key
```

Reply:
🤦‍♂ riight, it's that kind of morning. Thanks!